content_schemas/dist/formats/worldwide_organisation/frontend/schema.json

```diff
@@ -272,6 +272,14 @@
           "description": "Contact details for this Worldwide Organisation",
           "$ref": "#/definitions/frontend_links"
         },
+        "ordered_non_primary_role_people": {
+          "description": "People currently appointed to non-primary roles in this Worldwide Organisation",
+          "$ref": "#/definitions/frontend_links"
+        },
+        "ordered_primary_role_people": {
```

Comment:
Sorry, I've just read your Slack comment about this; Um, if its singular for now, I would go with singular. It will make more sense in Whitehall. If in the future, there is a decision to have multiple primary role people, they can change the name of the key then?

```ruby
def primary_role_person
   [@item.primary_role_people.first] # Or similar
end
```

Reply:
I've gone back and forth over this, and ended up on having three links for each type of role, in case the distinction between secondary role and office roles ever becomes relevant.

I've also gone with your suggestion of singular, because it looks like this is indeed the reality of the domain space (there can be only one!), but links have to be arrays so that's why the apparently unnecessary arraying.

Reply:
I think that makes sense 👍